client/public/index.html

```diff
@@ -24,7 +29,7 @@
       work correctly both with client-side routing and a non-root public URL.
       Learn how to configure a non-root public URL by running `npm run build`.
     -->
-    <title>CRA-With-API</title>
+    <title>NEOWISE-Project</title>
```

Comment:
You have an eye for details. That's great!  🚀

Reply:
thanks! 😃